src/UKHO.ERPFacade.Common/Logging/EventIds.cs

```diff
@@ -0,0 +1,23 @@
+﻿using Microsoft.Extensions.Logging;
+
+namespace UKHO.ERPFacade.Common.Logging
+{
+    public enum EventIds
+    {
+        /// <summary>
+        /// 920001 - An unhandled exception occurred while processing the request.
+        /// </summary>
+        UnhandledControllerException = 920001,
+        /// <summary>
+        /// 920002 - ERP Facade Request event started.
+        /// </summary>
+        ErpFacadeApiCalled = 920002
+    }
+    public static class EventIdExtensions
```

Comment:
Please move this into a separate file EventIdsExtensions.cs please

Reply:
Moved